### PR TITLE
 Bring back 2.10 but keep 2.13.0-M3

### PR DESCRIPTION
**Pull request by BennyHill**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -19,13 +19,17 @@ lazy val commonSettings = Seq(
...
-    "org.specs2" %%% "specs2-core"       % specs2Version.value % "optional",
-    "org.specs2" %%% "specs2-scalacheck" % specs2Version.value % "optional",
-    "org.scalacheck" %%% "scalacheck" % "1.13.5",
-    "org.scalatest"  %%% "scalatest"  % "3.0.5-M1" % "optional"
+    "org.scalacheck" %%% "scalacheck" % "1.13.5"
   ),
+  libraryDependencies += {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 13)) =>
+        "org.scalatest" %%% "scalatest" % "3.0.5-M1"
```
Comment (larsrh) [inline build.sbt]: Is there a reason why Scalatest isn't optional anymore?

Comment (BennyHill) [reply build.sbt]: 'cos I copied this from catalysts...fixing now :wink:

**Comment by larsrh** [comment]
Thank you, @BennyHill! Release is running.

**Comment by BennyHill** [comment]
Awesome, and thanks to you as well, @larsrh :+1:

**Pull request merged**